src/devices/schneider_electric.ts

```diff
@@ -15,6 +15,9 @@ const ea = exposes.access;
 const exposesLocal = {
     indicator_mode: e.enum('indicator_mode', ea.ALL, ['consistent_with_load', 'reverse_with_load', 'always_off', 'always_on'])
         .withDescription('Led Indicator Mode'),
+    switch_action: e.enum('switch_action', ea.ALL, ['light', 'light_opposite', 'dimmer', 'dimmer_opposite', 'standard_shutter',
```

Comment:
Instead of adding the expose/fromZigbee/toZigbee converter, can you use the new `enumLookup` instead? Example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/aeea2fc1b056c61c22e0bcd876e383825c883ccb/src/devices/custom_devices_diy.ts#L1305

This reduces code and improves consistency.

Reply:
I changed it, hope i did it correct as i was unable to test because i don't know how to import `modernExtend` into my custom converter.
I also updated the `indicator_mode`

Reply:
do `const {enumLookup} = require('zigbee-herdsman-converters/lib/modernExtend');

Reply:
thanks - so doesn't quite work yet i always get

```
Error: Write 0x540f57fffe306b17/3 clipsalWiserSwitchConfigurationClusterServer({"1":{"value":1,"type":48}}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":4190,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'UNSUPPORTED_ATTRIBUTE')
    at Endpoint.checkStatus (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:319:28)
    at Endpoint.write (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:392:22)
    at Object.toZigbee.convertSet (/app/node_modules/zigbee-herdsman-converters/src/lib/modernExtend.ts:399:13)
    at Publish.onMQTTMessage (/app/lib/extension/publish.ts:249:36)
    at EventEmitter.wrappedCallback (/app/lib/eventBus.ts:174:17)
```

Can't really figure out the issue. Is there any way to activate verbose logging so i can see what z2mqtt sends when i use the old code - because it worked even though it didn't "look" as good.

Reply:
Also the `endpoint` parameter of `enumLookup` seems to behave weird because when i add it i get the error
`No converter available for 'switch_action_21' ...`.
Is it possible to use something as `withEndpoint` with `enumLookup` ? Else i would need to define it multiple times for different endpoints.

Reply:
So it's the wrong endpoint i think - how do i set the correct endpoint to `enumLookup`?

Reply:
sorry found my issue i was missing `meta: {multiEndpoint: true},` and 
```
endpoint: (device) => {
        return {
            'l3': 3, 'l21': 21, 'l2': 22,
        };
    },
```